packages/compass-app-stores/src/stores/instance-store.js

```diff
@@ -23,6 +23,10 @@ function getTopologyDescription(topologyDescription) {
 
 const store = createStore(reducer);
 
+store.getInstance = () => {
+  return store.getState().instance;
+};
```

Comment:
TODO: we have more places where we access instance, change them to use this method